dependency-check-suppressions.xml

```diff
@@ -1,111 +1,53 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
-    <suppress>
-        <notes>CVE-2018-1258: we do not use Spring Method Security, so this does not affect us. see https://pivotal.io/security/cve-2018-1258.
-            we are on latest Spring Framework version : 5.2.2.RELEASE : https://github.com/spring-projects/spring-framework/wiki/Spring-Framework-Versions</notes>
+    <suppress until="2021-01-25">
```

Comment:
As discussed please do the clustering of conditions and add appropriate comments.

Reply:
supressions are grouped and comments are added